nullbot.py

```diff
@@ -20,6 +19,13 @@ class NullBot:
         # Do an initial sync and ignore it, to throw out old messages
         await self.client.sync()
 
+        try:
+            self.pgc = await asyncpg.connect()
+        except e:
+            print("Cannot connect to database!")
+            print(e)
+            exit(1)
```

Comment:
replace these two lines with `raise`, or remove the try/except block entirely and let the exception propagate normally